src/components/Transaction/Header.tsx

```diff
@@ -44,6 +45,9 @@ export const TransactionHeader = ({
         txState === TxState.LoadingTxDetails ||
         txState === TxState.MPoolPushing) && <LoaderGlyph />}
     </StandardBox>
+    {warning && txState < TxState.LoadingTxDetails && (
+      <WarningBox>{warning}</WarningBox>
+    )}
```

Comment:
Thinking through the UX on this... are we sure this is the right state to display the warning? I'd probably vote for one of the following:

1. Always show the warning if there is one (state agnostic)
2. Show the warning during the transaction review state

Reply:
Good call! I think 2 would be good UX, let's try it out and see how it feels in the safe.